index.js

```diff
@@ -1,135 +1,35 @@
 #! /usr/bin/env node
-const path  = require('path');
-const fs    = require('fs');
 const yargs = require('yargs');
-const chalk = require('chalk');
 
-const Select    = require('./lib/test/select');
-const Operators = require('./lib/infra/operators');
-const Steps     = require('./lib/read/stepsReader');
+const Stepper = require('./lib/read/stepper');
+const Reporter = require('./lib/read/reporter');
 
-(async()=>{
+(async () => {
 
-    yargs.command('test <stepfile>', 'Test markdown file', (yargs) => { }, async (argv) => {
-
-        await testreport("test", argv);
-
-    });
-
-    yargs.command('report <stepfile>', 'Test markdown file and report feedback into rendered output', (yargs) => { }, async (argv) => {
-
-        await testreport("report", argv);
-
-    });
-
-    yargs.command('notebook <notebook>', 'Test notebook html and report feedback into rendered output', (yargs) => { }, async (argv) => {
-        await notebook(argv);
+    yargs.command('report <doc> [html]', 'Test markdown/steps file and report feedback into rendered output', (yargs) => { }, async (argv) => {
```

Comment:
Having `[html]` optional positional argument is useful for cases where you have another way of retrieving html externally.

Suggest adding an option flag for specifying output report name/location.

Reply:
added in ffcf2ce.